src/client/src/translations/whitelist_en.json

```diff
@@ -68,7 +68,6 @@
   "cosmosResourceModule.saveChanges",
   "cosmosResourceModule.createCosmosRes",
   "cosmosResourceModule.internalName",
-  "footer.license",
```

Comment:
why was this line removed?

Reply:
I'm guessing this was due to running build since footer.license isnt used anywhere else but I may be wrong. This shouldn't be removed technically, but it reveals an underlying issue https://github.com/microsoft/WebTemplateStudio/blob/c3eeff139a294756156842a7cf394d254d500f34/src/client/src/containers/Footer/index.tsx
Looking at this file it seems that footer.license isnt used anywhere, it should be doing similar to what the aria label is doing with intl.

Reply:
@njindal239 why was the line removed

Reply:
This change comes in when I run ./build on the project. I am not sure whether it needs to be there or not but since it came out of building the project, I committed it with my changes.